### DISCUSSION TITLE
Security advisory credit badge not appearing on profile?

### DISCUSSION THREAD

**Discussion by CopperEagle** [community/community#124166] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello there!

About a month ago, I reported a vulnerability and the credit now appears in the [advisory](https://github.com/advisories/GHSA-g7vv-2v7x-gj9p). However, I am wondering why there is no security advisory badge on my profile ("highlights" section), as is mentioned in the [docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile). It doesn't look like this feature has been removed, as seen [here](https://github.com/tomaarsen).

There have also been [similar problems](https://github.com/orgs/community/discussions/41666) in the past and seemingly no automated solution. Does Github staff add this manually?

Any help with this would be awesome! Thanks so much!

**Comment by abdemeh** [comment]
Here are some steps you can take and considerations to keep in mind:
- Verify Credit in the Advisory
- Check Badge Criteria
- Delay in Badge Display
- Manual Addition by GitHub Staff
- Contact GitHub Support

**Reply by CopperEagle** [reply]
Thank you for your reply!

What do you mean by verifying? I accepted credit quite a while ago and it does show up in the [advisory](https://github.com/advisories/GHSA-g7vv-2v7x-gj9p). In the advisory, it shows the disclaimer below, too. So the system did register it - it just does not appear on the profile.
![Profile card](https://github.com/community/community/assets/137458753/accd8080-18eb-478d-9fea-2d6d48911988)


In the description above, I have linked an account that meets the same criteria as I do (accepted credit for reporting a vulnerability) and it does show up there. 

Edit: Moving it over to be a reply.
